flink-runtime/src/main/scala/org/apache/flink/runtime/ActorLogMessages.scala

```diff
@@ -30,13 +30,15 @@ trait ActorLogMessages {
     override def isDefinedAt(x: Any): Boolean = _receiveWithLogMessages.isDefinedAt(x)
 
     override def apply(x: Any):Unit = {
-      if(log.isDebugEnabled) {
-        log.debug(s"Received message ${x} from ${self.sender}.")
```

Comment:
Ah, that got lost while optimizing code paths. I will add it back in.
Thanks for catching it.